customers/src/customers/service/services/customer.service.ts

```diff
@@ -16,10 +17,12 @@ export class CustomerService {
 
   public async create(email: string): Promise<Customer> {
     const customer = await this.repository.create(email);
-    await this.eventProducer.sendEvent(config.messageBroker.topics.customersTransaction, {
+    const event = {
       data: { email },
       eventType: SystemEventType.CustomerCreated,
-    });
+    };
+    await this.eventProducer.sendEvent(config.messageBroker.topics.customersTransaction, event);
+    events_sent_total.inc({ topic: config.messageBroker.topics.customersTransaction, event: JSON.stringify(event) });
```

Comment:
In metrics, you shouldn't send a lot of unique values to labels. In your case, you're also sending the body of the entity, which I think is a bit much. Metrics are designed for simple data.